app/src/androidTest/java/com/github/se/cyrcle/ui/authentication/SignInTest.kt

```diff
@@ -62,7 +62,7 @@ class LoginTest {
     composeTestRule
         .onNodeWithTag("LoginTitle")
         .assertIsDisplayed()
-        .assertTextEquals("Welcome to Cyrcle")
+        .assertTextEquals("Welcome to")
```

Comment:
Using this logic you can just write Welcome

Reply:
Even better: Welcome!

Reply:
As i said in the RP, the logo says "Cyrcle", and since day 1 it bothered me that it said "Welcome to Cyrcle Cyrcle" with the logo